uiowa/uiowa
# Rename all site splits to `site` and move to `config/sites` directory.

Resolves #2359 

### Testing
1. Inspect file changes tab. Although many files were changed, most were just renames to a new directory.
2. Add the following to `blt/local.blt.yml`:
```
  - admissions.uiowa.edu
  - brand.uiowa.edu
  - coronavirus.uiowa.edu
  - genetics.grad.uiowa.edu
  - grad.uiowa.edu
  - hr.uiowa.edu
  - icsa.uiowa.edu
  - ighn.international.uiowa.edu
  - immuno.grad.uiowa.edu
  - iowasuperfund.uiowa.edu
  - pharmacy.uiowa.edu
  - policy.clas.uiowa.edu
  - presidentialsearch.uiowa.edu
  - sitenow.uiowa.edu
  - sppa.uiowa.edu
  - tippie.uiowa.edu
  - transportation.uiowa.edu
  - uiowa.edu
```

3. Run `blt dsa` and closely examine the output for each site. There should be no changes outside of the usual local environment split **besides** the deletion of the previous site split. For example, admissions lists the following:
```
...
 [notice] Synchronized extensions: install config_inspector.
 [notice] Synchronized extensions: install reroute_email.
 [notice] Synchronized extensions: install update.
 [notice] Synchronized extensions: install views_ui.
 [notice] Synchronized extensions: install devel.
 [notice] Synchronized extensions: install stage_file_proxy.
 [notice] Synchronized extensions: uninstall acquia_purge.
 [notice] Synchronized extensions: uninstall siteimprove.
 [notice] Synchronized configuration: delete config_split.config_split.admissions.
 [notice] Synchronized configuration: create system.menu.devel.
 [notice] Synchronized configuration: create tour.tour.views-ui.
...
```

Look out for any modules that get uninstalled and reinstalled which would indicate a gap where both splits are inactive. 

Additionally, visit all these sites in your browser, log in and look for any visual regressions.

4. Unless you have untracked changes in the config directory from other work, you can run `git clean -fd config` because `dsa` results in the recreation of the old config directories. I think this happens because the old split is still active during the update and it recreates the directory.  I tried specifically disabling the old splits but that didn't work. This shouldn't happen once this is deployed to prod. 
5. Run `blt ume 'config-split:export site'` to test exporting all site splits.
